docs/source/changelog.rst

```diff
@@ -23,6 +24,7 @@ Changelog
         * Implement automated process for checking critical dependencies (:pr:`1045`, :pr:`1054`, :pr:`1081`)
         * Don't run changelog check for release PRs or automated dependency PRs (:pr:`1057`)
         * Fix non-deterministic behavior in Dask test causing codecov issues (:pr:`1070`)
+        * Remove xfail cases from ``test_plotting.py`` (:pr:`1086`)
```

Comment:
The main change is fixing `EntitySet.plot`, an additional changelog entry is unnecessary

Reply:
> an additional changelog entry is unnecessary

Not sure what you mean by this, should I remove the changes I made to testing section?

Reply:
> Not sure what you mean by this, should I remove the changes I made to testing section?

Sorry, I meant remove this line from the changelog, since we already have the line added in the Fixes section.